config/locales/en.yml

```diff
@@ -32,6 +32,108 @@
 en:
   i18n:
     direction: ltr
+    transliterate:
+      rule:
+        α: a
+        ά: a
+        Α: a
+        Ά: a
+        β: v
+        Β: v
+        γ: g
+        Γ: g
+        δ: d
+        Δ: d
+        ε: e
+        έ: e
+        Ε: e
+        Έ: e
+        ζ: z
+        Ζ: z
+        η: h
+        ή: h
+        Η: h
+        Ή: h
+        θ: th
+        Θ: th
+        ι: I
+        ί: I
+        Ι: I
+        Ί: I
+        κ: k
+        Κ: k
+        λ: l
+        Λ: l
+        μ: m
+        Μ: m
+        ν: n
+        Ν: n
+        ξ: x
+        Ξ: x
+        ο: o
+        Ο: o
+        ό: o
+        Ό: o
+        π: p
+        Π: p
+        ρ: r
+        Ρ: r
+        σ: s
+        Σ: s
+        ς: s
+        τ: t
+        Τ: t
+        υ: y
+        ύ: y
+        Υ: y
+        Ύ: y
+        φ: f
+        Φ: f
+        χ: ch
+        Χ: ch
+        ψ: ps
+        Ψ: ps
+        ω: w
+        ώ: w
+        Ώ: w
+        Ω: w
```

Comment:
These ones are the only ones that stand out as possibly wrong to me. I think Omega is usually a long "o" sound (like in "toe"), so it should probably transliterated as "o", not "w".

Reply:
From my extensive duolingo greek 😬 

I'm sure we could find a native speaker if we wanted to be sure.

Reply:
I also wondered about `ου` and `ευ` which if I remember correctly depend on the consonant that follows them — but I suppose it's close enough to be recognisable and it's pretty unambiguous.

Reply:
Good spot! I guess it makes little difference in practice though, as it's only used in base_path creation.

Reply:
Thanks for this, I'll trust your judgement and open a PR to change the letters for omega!

Reply:
https://github.com/alphagov/specialist-publisher/pull/1771